deployment/frontend/src/components/dashboard/_shared/Row.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import { EyeIcon, ChevronDownIcon, ChevronUpIcon } from '@heroicons/react/24/outline';
+
+
+type RowButton = {
+  label?: string;
+  icon?: React.ReactNode;
+  color?: string;
+  onClick: () => void;
+}
+
+type RowLinkButton = {
+  label?: string;
+  link?: string;
+}
+
+type RowProps = {
+  rowMain: React.ReactNode;
+  rowSub?: React.ReactNode;
+  isDropDown?: boolean;
+  controlButtons?: RowButton[];
+  linkButton?: RowLinkButton;
+  groupStyle?: string
+  className?: string;
+}
+
+export default function Row({ rowMain, rowSub, isDropDown, controlButtons, linkButton, groupStyle, className }: RowProps) {
```

Comment:
You should probably use a @headless/ui Disclosure element, also add a nice transition so it appears as an animation

Reply:
also the entire element should change color after click(even if its not hovered) you can check the DataFiles.tsx component 

Or just how you did the topics dropdown